packages/server/tests/localAcceptance.env

```diff
@@ -27,7 +27,7 @@ LIMIT_DURATION=90000
 SERVER_REQUEST_TIMEOUT_MS=60000
 MEMWATCH_ENABLED=true
 WRITE_SNAPSHOT_ON_MEMORY_LEAK=false
-HBAR_RATE_LIMIT_TINYBAR=5000000000# 50 HBARs
-HBAR_RATE_LIMIT_DURATION=80000# 80 seconds
-HBAR_RATE_LIMIT_BASIC=4000000000# 40 HBARs
+HBAR_RATE_LIMIT_TINYBAR=9000000000# 90 HBARs
+HBAR_RATE_LIMIT_DURATION=86400000# 24 hours
+HBAR_RATE_LIMIT_BASIC=6000000000# 60 HBARs
```

Comment:
**Note for Reviewers:** 
- The values of HBAR_RATE_LIMIT_TINYBAR has been increased due to the addition of more exhaustion tests in the hbar limiter. 

- Increasing HBAR_RATE_LIMIT_DURATION prevents the limiter from resetting during tests.

- Increasing HBAR_RATE_LIMIT_BASIC prevents api_batch1 from becoming flaky and encountering HBAR rate limit errors.